### ISSUE TITLE
issue about reproducing results on SNLI dataset 

### ISSUE THREAD

**Issue by 20000607-lxc** [ChunyuanLI/Optimus#24]
Hi! I'm trying to reproduce the reported result on SNLI, I followed the doc 'optimus_for_snli.md' and successfully downloaded the checkpoints, but when I run your examples, it turns out that in file run_latent_generation.py, the sample_sequence_conditional function receives 'input_ids' and 'past' in mismatched shape. I can fix this by past = torch.mean(past, dim=0).unsqueeze(0), but is it right? Thanks for reading.

**Comment by lxcccc-maker** [comment]
Hi! Another problem is that when running 'run_lm_vae_training.py' with Snli data you provided, I got extremely large perplexity = 4.017335065391466e+66, and  kl = 1922.1040397033692.

**Comment by 20000607-lxc** [comment]
Hi! Another problem is that when running 'run_lm_vae_training.py' with Snli data you provided, I got extremely large perplexity = 4.017335065391466e+66, and kl = 1922.1040397033692.
